### PR TITLE
Add list option for "minikube node" command

### LINKED ISSUE #7667: Add list option for "minikube node" command
with #94 adding support for multi-node kubernetes cluster with minikube, we need and option to list the nodes. Currently there is no "minikube node list" command.

minikube version: v1.9.2

current output:
minikube node list
💡  Usage: minikube node [add|start|stop|delete]

expected output:
"minikube node list" should list the node name, status and ip addresses

### PR DESCRIPTION
**Pull request by radeksm**
Add option to list the nodes.
So far there was no _**minikube node list**_ command.

fixes #7667

### CODE REVIEW

File: cmd/minikube/cmd/node_list.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+		if len(cc.Nodes) < 1 {
+			glog.Warningf("Did not found any Minikube node.")
+		} else {
+			glog.Infof("%v", cc.Nodes)
+		}
+
+		for _, n := range cc.Nodes {
+			machineName := driver.MachineName(*cc, n)
+			fmt.Printf("%s\n", machineName)
```
Comment (sharifelgamal) [inline cmd/minikube/cmd/node_list.go:50]: could you also list the node's IP address (n.IP) here as well?

Comment (radeksm) [reply cmd/minikube/cmd/node_list.go]: Added IP to the output, nothing fancy:

`[root@c8k20 radek]# ./minikube-linux-amd64  node list`
`minikube        192.168.122.29`

maybe some JSON format output in next PR?

**Comment by sharifelgamal** [comment]
Thank you for the contribution!

**Pull request merged**